### ISSUE TITLE
Incorrect flags for some countries

### ISSUE THREAD

**Issue by zhe1ka** [jackocnr/intl-tel-input#947]
Hi.
An incorrect flag for Ukraine - it should be as for Uganda (blue/yellow colors). I've checked for Uganda and it's incorrect as well :)
Also a position is a slight incorrect: for `.iti__flag.iti__ug { background-position: ` instead of `-5197px 0px;` it's better `-5195px 0px;`

![image](https://user-images.githubusercontent.com/6460571/62483562-a6b51580-b7c0-11e9-94fa-831f9e7305b0.png)

UPD: i noticed that all flags has an offset +1px.
this problem came to me after upgrading from v.15.*

**Comment by zhe1ka** [comment]
Actually positions is a slightly incorrect for a lot of flags.
P.S: Google chrome, MacOS

**Comment by vjmani** [comment]
Is there any resolution for this issue, I am also facing this issue in some instances PFB, getting wrong flag for Turkey:
![image](https://user-images.githubusercontent.com/7287117/62775716-12032e00-bac6-11e9-9049-2874ee004b8b.png)

**Comment by zhe1ka** [comment]
i found a solution for my case:

make sure to import .css file from the lib
`import 'intl-tel-input/build/css/intlTelInput.css';`

previously i copied css file and flags images. as I understand images were changed two slightly and therefore we got the problem. Now it works fine to me, for Turkey as well.

![image](https://user-images.githubusercontent.com/6460571/62850269-047abd80-bceb-11e9-8b75-0b5a9f899e87.png)
